Remotion/Web/Development.WebTesting/Configuration/Legacy/WebTestHostingAdapter.cs

```diff
@@ -42,6 +106,12 @@ public WebTestHostingAdapter (ProviderSettings providerSettings)
     public string Type => _providerSettings.Type;
 
     /// <inheritdoc />
-    public NameValueCollection Parameters => _providerSettings.Parameters;
+    public IReadOnlyDictionary<string, string> Parameters
```

Comment:
I'll admit, I expected this to end up as a copy operation.

Since the providesettings are private anyway, would a copy+cache of the copied value have been sufficient and resulted in less code? Since we want to kick it anyway and the code looks correct, I don't mind either way.

Reply:
Copy and cache would be out-of-date when somebody changes the `NameValueCollection`. As that is more likely than replacing the collection in its entirety I wrapped it in as a read-only. Probably this is already overkill but it wasn't that hard to implement so...